src/components/AccountsList/index.js

```diff
@@ -111,10 +112,11 @@ function AccountsList({
         {name && <Typography variant="h2">{name}</Typography>}
         {editable && (
           <Button variant="contained" color="primary" onClick={handleOpen}>
-            Add Account
+            {accountLabel}
```

Comment:
```suggestion
            {addAccountLabel}
```
... since we may have `deleteAccountLabel` as well.